### DISCUSSION TITLE
How to write my own scripts

### DISCUSSION THREAD

**Discussion by FeelLiao** [windingwind/zotero-actions-tags#438] [Q&A]
I want to write my own user scripts to power my study, but I do not know how to do it. The readme of this repository said that I could found more shared scripts in discussion, but there is little that fit my workflow. **I am here to ask for a guide to build the scripts.**

I want to know how to start a script as a beginner. Where to find the interface of Zotero such as automatic run the function provided by another plugin and getting information about a specifical item ?

**Comment by windingwind** [comment]
For beginners, you can starts with learning basic knowledge of JavaScript, then you'll be able to read the code of other plugins and understand how it works.
